simple/rdm_deferred_wq.c

```diff
@@ -628,7 +629,9 @@ int main(int argc, char **argv)
 		 tested_op == FI_OP_COMPARE_ATOMIC)
 		hints->caps |= FI_ATOMIC;
 
-	hints->mode = FI_CONTEXT | FI_LOCAL_MR;
+	hints->mode = FI_CONTEXT;
+	hints->domain_attr->mr_mode = (FI_MR_LOCAL | FI_MR_VIRT_ADDR |
+				       FI_MR_ALLOCATED);
```

Comment:
gurgle - this test should be updated to support provider keys too.  These changes look correct.